src/main/java/org/opensearch/performanceanalyzer/rca/framework/metrics/WriterMetrics.java

```diff
@@ -145,6 +154,15 @@
                     Statistics.SUM)),
 
     STALE_METRICS("StaleMetrics", "count", Arrays.asList(Statistics.COUNT)),
+
+    METRICS_WRITE_ERROR(
+            "MetricsWriteError",
+            "namedCount",
+            Collections.singletonList(Statistics.NAMED_COUNTERS)),
+
+    METRICS_REMOVE_ERROR("MetricsRemoveError", "count", Arrays.asList(Statistics.COUNT)),
+
+    METRICS_REMOVE_FAILURE("MetricsRemoveFailure", "count", Arrays.asList(Statistics.COUNT)),
```

Comment:
nit: Can we add documentation on what the metric denotes?

Reply:
Right, will add documentation for the metrics as part of https://github.com/opensearch-project/performance-analyzer-rca/pull/29